Kendo-UI-Bootstrap-Integration/src/Kendo-UI-Bootstrap-Integration/Models/Swatch.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace KendoUI_Bootstrap_Integration.Models
```

Comment:
Change the namespace to match the others used in the app - Kendo_UI_Bootstrap_Integration.Models